content/en/software/_index.md

```diff
@@ -4,18 +4,17 @@ type: docs
 aliases:
 
 ---
-This section contains information on installing or setting up Medley and using it.
+## Welcome to the Medley Interlisp Experience
 
-There are two main ways of running Medley: accessing it online in a web browser or installing and running the system on your computer. Online access is best for trying out Medley without installing anything, as well as for basic usage. Install the system on your computer if you want to run Medley offline, customize the environment, or simplify the exchange of files with Medley.
+The [Medley Interlisp Project](/project) has focused on bringing the [historical software](/history) to life. This section contains information on setting up the Medley system and using it.
 
-We are working on making an early version of Medley run in the browser. This example of software preservation will let you access a technology artifact of historical significance.
+There are two main ways of running Medley: accessing it online in a web browser or installing and running the system on your computer. Online access is good for trying out Medley without installing anything, as well as for basic usage. Install the system on your computer if you want to run Medley offline, customize the environment, or simplify the exchange of files with Medley.
```

Comment:
What does that last clause mean?  *Simplify the exchange of files with Medley.*  Do you mean loading files in to and out of the Medley system?

Reply:
I changed this to "Install the system on your computer if you want to run Medley offline, customize the environment, and have direct access to local files."

Some might consider the firewall of requiring explicit file transfer as an advantage of running online.  Might be a reason for using Docker, for that matter.